table/server/common/src/main/java/alluxio/table/common/BaseConfiguration.java

```diff
@@ -55,6 +57,23 @@ public String get(T property) {
     return value;
   }
 
+  /**
+   * Return the int value of this property , or the default value if the property is not defined.
+   *
+   * @param udbProperty the udb property to get the int value
+   * @return the int value of udb property
```

Comment:
"udb property" should just be "property"

Reply:
Fixed it.